### ISSUE TITLE
{Freeway-Bug} [Adaptive Cards> InputsWithValidation.json> Input.Toggle]: Voice access user is not able to access 'Start time' edit field through 'Move to Start time' command.

### ISSUE THREAD

**Issue by vaishali1397** [microsoft/AdaptiveCards#7521]
### Target Platforms

UWP

### SDK Version

3.0.2206.2001

### Application Name

Adaptive Cards

### Problem Description

**Problem Description**
Test Environment:
OS Version: 22H2 (Build 25126.1000)
Application: Adaptive Cards UWP
Application Version: 3.0.2206.2001
Tool: Voice access

**Repro Steps:**

1. Turn on voice access.
2. Open Adaptive Cards UWP application.
3. Command: Voice access wake up'
4. Command: Show Numbers
5. Select particular number for 'InputsWithValidation.json'
6. Command: Move to Start time
7. Observe the issue.

**Actual Result:**
Voice access user is not able to access 'Start time' edit field through 'Move to Start time' command instead it is accessible through 'Move to 16' command. 

**Expected Result:**
Voice access user should be able to access 'Start time' edit field through 'Move to Start time' command.

**User Impact:**
People with limited mobility (cerebral palsy, quadriplegia) who use Voice Access will get impacted as user will not be able to interact with the Start time edit field due to which they will not be able to make proper track of the controls.

**MAS reference link:**
[[No Disruption of Accessibility Features - Liquid (microsoft.com)]](https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/04.02.01#Zguide)

**Have feedback to share on Bugs?** Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-chisha%40microsoft.com%7c1334b110f98744a7f5df08d968eb842b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656183104980596%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=r4ndSHk/LrBDwzi7FO%2BjKN0DCnqUSUcELBvGRkgc5CU%3D&reserved=0)"

### Screenshots

https://user-images.githubusercontent.com/104483005/173436096-92b762f1-3d3f-40d6-b3a9-f9b94b85142c.mp4

<img width="960" alt="MAS4 3 1 Voice access user is not able to access Start time comboedit field" src="https://user-images.githubusercontent.com/104483005/173436306-7381b245-f7b9-42bd-afbf-116b0581e7aa.png">

### Card JSON

```JSON
NA
```

**Comment by vaishali1397** [comment]
Issue is repro'ing in below test environment, hence reactivating the bug.
Test Environment:
OS version: 22H2(Build 25272.1000)
Application: Adaptive Cards Visualizer (3.0.2210.26001 (latest) build.)

**Comment by JeanRoca** [comment]
This issue has been fixed and will be included in our next release. I am closing this issue for now. Once new version is available, if the issue persists please open a new issue to track.

**Comment by vaishali1397** [comment]
Issue has been fixed in below test Environment, hence Adding A11yTTValidated tag.
**Test Environment:**
OS Version: 22H2 (Build 25314. 1010)
Application: Adaptive Card Visualizer (Version 3.2.2303.22003)
**PFA for Reference:**
https://user-images.githubusercontent.com/104483005/228221572-b7057514-d403-41c4-978d-24e6900ccc1e.mp4
